src/main/java/net/minecraftforge/common/config/IFieldWrapper.java

```diff
@@ -68,6 +68,10 @@
      * @param reqMCRestart True, if a change in this property requires a restart of Minecraft
      * @param reqWorldRestart True, if the world needs to be reloaded after changes to this property
      */
+    default void setupConfiguration(Configuration cfg, String desc, String langKey, boolean reqMCRestart, boolean reqWorldRestart) {
+        return setupConfiguration(cfg, desc, langKey, reqMCRestart, reqWorldRestart, false);
+    }
+    
```

Comment:
The javadocs should be on this one with an @see on the other. Or just on both.

Reply:
Additionaly, mark it for removal in the next large update. That is our chance to unify these two methods.